### DISCUSSION TITLE
Exported jar file is huge i.e. > 200 MB

### DISCUSSION THREAD

**Discussion by ajaytiwa** [MuleSoft-AI-Chain-Project/mule-ai-chain-connector#72] [Q&A] [answered]
Anyone able to deploy the jar which has mac connectors into standalone runtime?
I have a simple flow .. exported jar of this simple flow goes up around 300 MB.
<img width="448" alt="image" src="https://github.com/user-attachments/assets/d2e7da75-ec41-46cf-9c55-5a6db1223ddf">

As per below jar should not be more then 200 MB 
https://help.salesforce.com/s/articleView?id=001120905&language=en_US&type=1

**Comment by amirkhan-ak-sf** [comment]
Hi @ajaytiwa 

you can deploy it manually into the Standalone Runtime. The limitations are if you deploy via Runtime Manager UI. Hope this helps.

**Reply by ajaytiwa** [reply, marked as answer]
@amirkhan-ak-sf Thanks I will try to deploy ..
